qiita_ware/ebi.py

```diff
@@ -704,8 +704,10 @@ def write_xml_file(self, element, fp):
         fp : str
             The filepath to which the XML will be written
         """
-        makedirs(self.xml_dir)
-        ET.ElementTree(element).write(fp, encoding='UTF-8')
+        if not exists(self.xml_dir):
```

Comment:
I have a util function in qiita_db that wraps this called create_nested_path(). I think this is fine for now. I'll make a note to migrate that functionality so it's accessible in both locations and replace this line.

Reply:
Sounds good!!